docs/topics/api/overview.rst

```diff
@@ -246,6 +249,25 @@ specified.
 
 .. _api-versions-list:
 
+
+-----------
+Site Status
+-----------
+
+.. _`api-site-status`:
+
+This special endpoint just returns if the site is in read only mode, and if there is a site notice currently in effect.
+See :ref:`maintainance mode <api-overview-maintainance>` for more details of when the site is read only and how requests are affected.
+
+
+.. http:get:: /api/v4/site/
+
+    .. _site-status-object:
+
+    :>json boolean read_only: Is the site in read-only mode.
+    :>json string notice: A site-wide notice about any current known difficulties or restrictions.  If this API is being consumed by a tool/frontend it should be displayed to the user.
```

Comment:
This should probably return `null` when it is not set, no?

Reply:
https://github.com/mozilla/addons-server/pull/12356/commits/4acd93a499ade786efc4967b45807f2550d4a248